src/app.ts

```diff
@@ -1,42 +1,28 @@
-import express, { Express } from "express";
-
-import bodyParser from "body-parser";
+import express, { Express, json, urlencoded } from "express";
 import path from "path";
 import cors from "cors";
+import helmet from "helmet";
+import morgan from "morgan";
```

Comment:
What are thoseee

Reply:
In general:
1. `helmet` is the most popular (almost mandatory) server-side security library, if you're going to whiten this, it's a must-have.
2. `morgan` is the most popular logging middleware for HTTP requests made by your code. Once you run your code, it logs all the info about your requests directly to your console

Reply:
How do you actually use them in the app? Is importing them enough to accomplish something?